console/apps/web-ui/public/config.js

```
@@ -18,41 +18,35 @@
-    },
-    tokenLifecycle: {
-      refreshToken: {
-        autoRefresh: true,
+        validate: '' === 'true',
+        clockTolerance: Number('') || 300,
       },
     },
     storage: 'localStorage',
   },
-  disableAuth: false,
+  disableAuth: 'true' === 'true',
```

Biome (2.4.15)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)